api/server/docker.go

```diff
@@ -651,6 +651,8 @@ func (d *driver) mount(w http.ResponseWriter, r *http.Request) {
 
 	// get spec and name from request
 	_, spec, _, _, name := d.SpecFromString(request.Name)
+	_, _, _, _, volID := d.SpecFromString(request.ID)
```

Comment:
You don't need this. Just use the provided ID as is.

Reply:
OK, I'll change that